src/routes/activityReports/handlers.js

```diff
@@ -40,8 +40,9 @@ const logContext = {
 
 async function sendActivityReportCSV(reports, res) {
   const csvRows = await Promise.all(reports.map((r) => activityReportToCsvRecord(r)));
+
   const csvData = stringify(
-    csvRows,
+    validateCsvRows(csvRows),
     {
       header: true,
       quoted: true,
```

Comment:
can we add destructuring on line 421 in this file to enable filtering
Suggestion:
Change `{ readRegions, limit: null },`
to `{ ...readRegions, limit: null },`